bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/transport/MiIoAsyncCommunication.java

```diff
@@ -191,12 +195,11 @@ MiIoSendCommand sendMiIoSendCommand(MiIoSendCommand miIoSendCommand) {
     }
 
     public synchronized void startReceiver() {
-        if (senderThread == null) {
+        MessageSenderThread senderThread = this.senderThread;
+        if (senderThread == null || !senderThread.isAlive()) {
             senderThread = new MessageSenderThread();
         }
-        if (!senderThread.isAlive()) {
-            senderThread.start();
-        }
```

Comment:
If you create a new thread isn't it still supposed to be started here? right now you are starting the thread in the constructor (and elsewhere maybe, I didn't check too thoroughly).